controller/space_test.go

```diff
@@ -16,25 +16,49 @@ import (
 func TestDeleteOpenShiftResource(t *testing.T) {
 
 	t.Run("ok", func(t *testing.T) {
-		// given
-		r, err := recorder.New("../test/data/deployments/deployments_delete_space.ok")
-		require.NoError(t, err)
-		defer r.Stop()
-
-		spaceID, err := goauuid.FromString("aec5f659-0680-4633-8599-5f14f1deeabc")
-		require.NoError(t, err)
-		ctx := context.Background()
-		config, err := configuration.New("")
-		require.NoError(t, err)
-
-		client := &http.Client{
-			Transport: r.Transport,
-		}
-
-		// when
-		err = deleteOpenShiftResource(client, config, ctx, spaceID)
-		// then
-		require.NoError(t, err)
+		t.Run("with delete pipeline success", func(t *testing.T) {
+			// given
+			r, err := recorder.New("../test/data/deployments/deployments_delete_space.ok")
```

Comment:
it's better to have separate dir for pipeline rather using `deployments` dir to keep test files.